packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -78,6 +81,9 @@ BuildRequires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.14.1
 BuildRequires: %{?scl_prefix}rubygem(foreman_remote_execution) >= 3.0
 BuildRequires: %{?scl_prefix}rubygem(dynflow) >= 1.2.0
 BuildRequires: %{?scl_prefix}rubygem(activerecord-import)
+%if 0%{?rhel} == 7
+BuildRequires: %{?scl_prefix}rubygem(qpid_messaging)
+%endif
```

Comment:
Given that this is in the auto-generated section but has a conditional around it, I think you should move this and the Requires to https://github.com/theforeman/foreman-packaging/pull/6200/files#diff-ea1969cc98962976d79819fceee9455b08e0f99c44e5071fa281f3718bb8e3f8R27

Reply:
Related, see https://github.com/Katello/katello/pull/9088/files#r561227938

Reply:
Good call!

Reply:
@ehelms updated!